zaza/openstack/charm_tests/ovn/tests.py

```diff
@@ -50,6 +62,19 @@ def test_pause_resume(self):
             logging.info('Testing pause resume (services="{}")'
                          .format(self.services))
 
+    def test_nrpe_configured(self):
+        """Confirm that the NRPE service check files are created."""
+        units = zaza.model.get_units(self.application_name)
+        cmds = []
+        for check_name in self.nrpe_checks:
+            cmds.append(
+                'egrep -oh /usr/local.* /etc/nagios/nrpe.d/'
+                'check_{}.cfg'.format(check_name)
+            )
+        ret = self._retry_check_commands_on_units(cmds, units)
+        logging.info(ret)
```

Comment:
This produces a test output like this on success:
```
06:38:29 2020-10-01 06:38:29 [INFO] Confirm that the NRPE service check files are created.
06:38:29 2020-10-01 06:38:29 [INFO]  ... 
06:38:31 2020-10-01 06:38:31 [INFO] None
06:38:31 2020-10-01 06:38:31 [INFO] ok
```
And that may be a bit terse and/or confusing for someone not familiar with how the test works.

Would logging only when `ret` is not None be an alternative?

Reply:
I think that's an excellent plan.